src/services/OktaApiService.php

```diff
@@ -28,6 +28,7 @@ public function buildAuthorizeUrl($state)
             'client_id' => $this->clientId,
             'redirect_uri' => $this->redirectUri,
             'state' => $state,
+            'scope' => 'openid',
```

Comment:
Thanks for this!

Reply:
@aaronpk I thought it should work too, but here is what happens without the two requires even after a `composer install`.  Here is the output of `composer install` and `php -S 127.0.0.1:8080 -t public`:
```
cberkstresser@apollo:~/okta-php-core-login-example$ composer install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Nothing to install or update
Generating autoload files
cberkstresser@apollo:~/okta-php-core-login-example$ php -S 127.0.0.1:8080 -t public
[Mon Aug 31 12:03:14 2020] PHP 7.4.3 Development Server (http://127.0.0.1:8080) started
[Mon Aug 31 12:03:23 2020] 127.0.0.1:38286 Accepted
[Mon Aug 31 12:03:24 2020] PHP Fatal error:  Uncaught Error: Class 'Src\Services\OktaApiService' not found in /home/cberkstresser/okta-php-core-login-example/public/index.php:7
Stack trace:
#0 {main}
  thrown in /home/cberkstresser/okta-php-core-login-example/public/index.php on line 7
[Mon Aug 31 12:03:24 2020] 127.0.0.1:38286 [500]: GET / - Uncaught Error: Class 'Src\Services\OktaApiService' not found in /home/cberkstresser/okta-php-core-login-example/public/index.php:7
Stack trace:
#0 {main}
  thrown in /home/cberkstresser/okta-php-core-login-example/public/index.php on line 7
[Mon Aug 31 12:03:24 2020] 127.0.0.1:38286 Closing
[Mon Aug 31 12:03:24 2020] 127.0.0.1:38296 Accepted
[Mon Aug 31 12:03:24 2020] 127.0.0.1:38296 [404]: GET /favicon.ico - No such file or directory
[Mon Aug 31 12:03:24 2020] 127.0.0.1:38296 Closing
```

Reply:
okay let me take a closer look...